mahmud-shna/index.html

```diff
@@ -0,0 +1,71 @@
+            <div class="footer">
+              <table >
+                <thead>
+                  <tr>
+                    <td width="100px">About</td>
+                    <td width="100px">Twitter</td>
+                    <td>Github</td>
+                  </tr>
+                </thead>
+              </table>
```

Comment:
this table is actually a  navbar and you should put it into footer tag outside the main tag